### PR TITLE
ARM support

### PR DESCRIPTION
**Pull request by alvarocabanas**
- Add ARM support

### CODE REVIEW

File: build/.goreleaser.yml [hunk 1]
```diff
@@ -30,17 +30,19 @@ builds:
...
     description: "New Relic Infrastructure jmx Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from jmx components."
     license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
-    
+
+    builds:
+      - nri-jmx-nix
+
     dependencies:
       - "newrelic-infra"
+      - "nrjmx"
```
Comment (roobre) [inline build/.goreleaser.yml:45]: Seems to collide with https://github.com/newrelic/nri-jmx/pull/79, I'd wait until that's merged

**Review by varas — CHANGES_REQUESTED** [review]
https://github.com/newrelic/nri-jmx/pull/80/files#r649331274 dep should be >= 1.5.3

File: build/.goreleaser.yml (hunk 1 above)
Comment (alvarocabanas) [reply build/.goreleaser.yml:45]: Yes, I am aware, I will wait until that is merged to rebase and get the >= 1.5.3

Comment (alvarocabanas) [reply build/.goreleaser.yml]: I have rebased this branch and now it has the changed done by Cristian.

**Review by cristianciutea — APPROVED** [review]

**Pull request merged**